extensions/typescript-language-features/package.json

```diff
@@ -953,6 +954,11 @@
           }
         ]
       }
+    ],
+    "typescriptServerPlugins": [
+      {
+        "name": "typescript-vscode-sh-plugin"
```

Comment:
Does this plugin check the TS version before enabling itself? I think it would make sense if it only enabled itself on TS 3.8+

Reply:
add version check